### PR TITLE
Added fixed frame, nav colors, and initalpose tool

### PR DESCRIPTION
**Pull request by kayak0806**

### CODE REVIEW

File: src/dynamic_rviz_config/__init__.py [hunk 1]
```diff
@@ -49,9 +49,16 @@ def set_tool_topic(self, name, topic):
             if m.get('Class', '')==name:
                 m['Topic'] = topic
                 return
+    def set_fixed_frame(self,frame='map'):
+        v = self.get_visualization()
+        if 'Global Options' not in v:
+            v['Global Options'] = {'Fixed Frame':frame}
```
Comment (DLu) [inline src/dynamic_rviz_config/__init__.py:55]: What about if 'Global Options' is in v?

File: src/dynamic_rviz_config/__init__.py [hunk 2]
```diff
@@ -49,9 +49,16 @@ def set_tool_topic(self, name, topic):
...
 
-    def set_goal(self, topic):
+    def set_goal(self, topic='goal'):
         self.set_tool_topic('rviz/SetGoal', topic)
+
+    def set_initial_pose(self,topic='initialpose'):
```
Comment (DLu) [inline src/dynamic_rviz_config/__init__.py]: Missing space after comma

**Comment by DLu** [comment]
Addresses #1

File: src/dynamic_rviz_config/__init__.py (hunk 1 above)
Comment (kayak0806) [inline src/dynamic_rviz_config/__init__.py:55]: added an else statement that will just change the fixed frame if global options already exists

**Comment by DLu** [comment]
Thanks!

**Pull request merged**